arduino/cores/packagemanager/install_uninstall.go

```diff
@@ -395,3 +411,27 @@ func (pme *Explorer) IsToolRequired(toolRelease *cores.ToolRelease) bool {
 	}
 	return false
 }
+
+// RunPostInstallScriptTool runs the post_install.sh (or post_install.bat) script for the
+// specified toolRelease.
+func (pme *Explorer) RunPostInstallScriptTool(toolRelease *cores.ToolRelease) error {
+	if !toolRelease.IsInstalled() {
+		return errors.New(tr("tool not installed"))
+	}
```

Comment:
This check can be dropped (if the tool is not installed it will skip post_install anyway because the file doesn't exist).
Also dropping the check helps to combine the two methods `RunPostInstallScriptTool` and `RunPostInstallScript` as explained in my other comment.